internal/oas/schema.go

```diff
@@ -11,6 +11,9 @@ const (
 	Number  SchemaType = "number"
 	String  SchemaType = "string"
 	Boolean SchemaType = "boolean"
+
+	// Stream is custom type for raw byte stream.
+	Stream SchemaType = "x-stream"
```

Comment:
SchemaType contains types of an OpenAPI v3.0.3 JSON Schema.
```Stream``` it's not a part of that schema.
I think it should be detected and handled in ```gen``` pkg, not in the openapi parser.

Reply:
Agree, will extract logic to gen.

Reply:
Remove `stream` from `oas` package